extraction/src/main/java/io/ssc/trackthetrackers/extraction/resources/ResourceExtractor.java

```diff
@@ -99,7 +99,8 @@
           ParserRunner.ParseResult parseResult = javascriptParser.parse(scriptContents);
           findUrlCandidates(parseResult.ast, javaScriptUrlCandidates);
         }
-      } catch (Exception e) {}
+      } catch (Exception e) {
```

Comment:
we can leave the braces on the same line, we don't get anything from the extra line